bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -185,9 +193,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             + "<BinaryState>" + binaryState + "</BinaryState>" + "</u:SetBinaryState>" + "</s:Body>"
                             + "</s:Envelope>";
 
-                    String wemoURL = getWemoURL(remoteHost, "basicevent");
-                    if (wemoURL != null) {
-                        wemoCall.executeCall(wemoURL, soapHeader, content);
+                    if (host != null) {
+                        String wemoURL = getWemoURL(host, "basicevent");
+                        if (wemoURL != null) {
+                            wemoCall.executeCall(wemoURL, soapHeader, content);
```

Comment:
If we don't reach this line, this should be considered an error like the `catch` below logging failure to send command. Consider also moving it up as there is no reason to build request if we don't have a URL to target anyway.

Reply:
Done